ui/src/app/store/user/effects.ts

```diff
@@ -72,7 +75,14 @@ export class UserListEffect {
         this.actions.pipe(
             ofType(DeleteUser),
             switchMap(action => forkJoin([
-                this.userService.delete(action.organizationId, action.userId, action.newOwner),
+                this.userService.delete(action.organizationId, action.userId, action.newOwner).pipe(
+                  map(() => {
+                    this.store.dispatch(
```

Comment:
you should not dispatch action here. Needs to return an array of 2 new actions: DeleteUserSuccess and loadApplications (should be refactored to LoadApplications)